tests/io/MemoryHandleTest.php

```diff
@@ -16,6 +16,22 @@
 
 // @oss-disable: <<Oncalls('hack')>>
 final class MemoryHandleTest extends HackTest {
+  public async function testWriteOverwriteOutOfBound(): Awaitable<void> {
+    $handle = new IO\MemoryHandle('f', IO\MemoryHandleWriteMode::OVERWRITE);
+
+    await $handle->writeAllAsync('Hello, World!');
+
+    expect($handle->getBuffer())->toBeSame('Hello, World!');
+  }
+
+  public async function testWriteAppendOutOfBound(): Awaitable<void> {
+    $handle = new IO\MemoryHandle('f', IO\MemoryHandleWriteMode::APPEND);
+
+    await $handle->writeAllAsync('Hello, World!');
+
+    expect($handle->getBuffer())->toBeSame('fHello, World!');
```

Comment:
Could you change this to `expect($handle->getBuffer())->toEqual('fHello, World!');` instead?

It should work the same way: https://github.com/hhvm/fbexpect/blob/master/src/ExpectObj.hack#L40